packages/gatsby/src/query/graphql-errors.d.ts

```diff
@@ -0,0 +1 @@
+type RelayGraphQLError = Error & { validationErrors?: Object }
```

Comment:
```suggestion
interface RelayGraphQLError extends Error {
  validationErrors?: Object;
}
```
should this be an interface here as it extends an object type?